docs/src/content/docs/es/reference/frontmatter.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Referencia de Frontmatter
+description: Una visión general de los campos de frontmatter predeterminados que admite Starlight.
+---
+
+Puedes personalizar individualmente las páginas Markdown y MDX en Starlight estableciendo valores en su frontmatter. Por ejemplo, una página regular podría establecer los campos `title` y `description`:
+
+```md
+---
+title: Acerca de este proyecto
+description: Aprende más sobre el proyecto en el que estoy trabajando.
+---
+
+¡Bienvenido a la página Acerca de!
+```
+
+## Campos de frontmatter
+
+### `title` (requerido)
+
+**tipo:** `string`
+
+Debes proporcionar un título para cada página. Este se mostrará en la parte superior de la página, en las pestañas del navegador y en los metadatos de la página.
+
+### `description`
+
+**tipo:** `string`
+
+La descripción de la página es usada para los metadatos de la página y será recogida por los motores de búsqueda y en las vistas previas de las redes sociales.
+
+### `editUrl`
+
+**tipo:** `string | boolean`
+
+Reemplaza la [configuración global `editLink`](/reference/configuration/#editlink). Establece a `false` para deshabilitar el enlace "Editar página" para una página específica o proporciona una URL alternativa donde el contenido de esta página es editable.
+
+### `head`
+
+**tipo:** [`HeadConfig[]`](/reference/configuration/#headconfig)
+
+Puedes agregar etiquetas adicionales a la etiqueta `<head>` de tu página usando el campo `head` del frontmatter. Esto significa que puedes agregar estilos personalizados, metadatos u otras etiquetas a una sola página. Similar a la [opción global `head`](/reference/configuration/#head).
+
+```md
+---
+title: Acerca de nosotros
+head:
+  # Usa una etiqueta <title> personalizada
+  - tag: titúlo
```

Comment:
The `tag` is the HTML element, so this should still be title for the `<title>` element:

```suggestion
  - tag: title
```